package.json

```diff
@@ -80,6 +82,7 @@
     "eslint-plugin-no-only-tests": "^2.4.0",
     "eslint-plugin-svelte3": "^2.7.3",
     "eslint-svelte3-preprocess": "^0.0.4",
+    "ethers": "^5.0.12",
```

Comment:
Can we avoid using `ethers` since we are pulling in the web3.js provider for walletconnect?

Reply:
done!

Reply:
We actually need ethers in ./scripts/deploy-dev-contracts.js, added it back. it's a dev dependency btw.